src/connectors/HttpConnector.ts

```diff
@@ -50,7 +50,6 @@ export default class HttpConnector extends BaseHttpConnector<
     )
 
     if (eventConfiguration) {
-      this.app.getLogger().info('Handling event')
```

Comment:
maybe replace with .debug() ?

Reply:
We could - but the log message itself adds nothing I think?

Reply:
I think there's more to it in this story than removing the `handling event` message
I think it's relevant to keep it in Debug, or you never know if something happens.
I would do a few things on this story (there are suggestions only):
- add an option to EventConfiguration (silent true/false, default to false)
- each connector options take silent true/false
- if silent => no log at all, else keeping the current info and add the event.id (e.g. `Handling event Slack/message/123456789`))